### ISSUE TITLE
Items for a new user should be for a new user

### ISSUE THREAD

**Issue by jcoyne** [projecthydra/projecthydra.github.io#53]
See this menu.

<img width="244" alt="screen shot 2017-04-24 at 5 56 20 pm" src="https://cloud.githubusercontent.com/assets/92044/25362086/77a37ef6-2917-11e7-814b-39156dd99445.png">

As the original author of "Dive Deeper into Hyrax" and "Design Patterns in Hyrax," I'm going to say that these are not resources for new users.  They *could* be stashed under development resources.  But they probably ought to be just links to the original slideshows.  In their present form, they are not suitable as text only resources, they were slides from a talk and are missing lots of context.

**Comment by jenlindner** [comment]
Yeah ... you make a very valid point. I'm going to think about this overnight, right now they provide some content not found elsewhere here, but, yeah, where they are is misleading.
